nym-wallet/package.json

```diff
@@ -35,7 +36,11 @@
     "@nymproject/react": "^1.0.0",
     "@nymproject/types": "^1.0.0",
     "@storybook/react": "^6.5.15",
-    "@tauri-apps/api": "^1.2.0",
+    "@tauri-apps/api": "^2.4.0",
+    "@tauri-apps/cli": "^2.4.0",
```

Comment:
`@tauri-apps/cli` should be defined as dev-dependency

Reply:
Fixed 👍